src/pages-and-resources/discussions/data/redux.test.js

```diff
@@ -13,7 +13,8 @@ import { legacyApiResponse, piazzaApiResponse } from '../factories/mockApiRespon
 
 // Helper, that is used to forcibly finalize all promises
 // in thunk before running matcher against state.
-const executeThunk = async (thunk, dispatch, getState) => {
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I think this can be moved to a `utils.js` rather than exporting & using a function from another test suit.